allure-report-face/src/main/less/blocks/testcase.less

```diff
@@ -71,4 +71,17 @@
 
 .fa-database {
   font-size: 14px;
+}
+
+.param-holder {
+  white-space: nowrap;
```

Comment:
we already have common class for that: https://github.com/allure-framework/allure-core/blob/master/allure-report-face%2Fsrc%2Fmain%2Fless%2Fbase.less#L17
Use it instead of new one.